timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -38,8 +40,17 @@ def run_sigma_rule(self, query, rule_name, tag_list = None):
         events = self.event_stream(
             query_string=query, return_fields=return_fields)
         for event in events:
-            event.add_tags(['sigma_{0:s}'.format(rule_name)])
+            ts_sigma_rules = event.source.get('ts_sigma_rule', [])
+            ts_sigma_rules.append(rule_name)
+            event.add_attributes({'ts_sigma_rule': list(set(ts_sigma_rules))})
+            ts_ttp = event.source.get('ts_ttp', [])
+            for tag in tag_list:
+                if tag.startswith(('attack.', 'car.')):
```

Comment:
can you add a comment about why we hav `attack.` and `car.` handled in a special way?